src/components/Header/Header.tsx

```diff
@@ -0,0 +1,60 @@
+import React, { FormEvent, useEffect, useRef } from 'react';
+import classNames from 'classnames';
+
+type Props = {
+  onAddTodo: (title: string) => void,
+  todoTitle: string,
+  setTodoTitle: (todoTitle: string) => void,
+  isBeingAdded: boolean,
+  onStatusAll: (completed: boolean) => void,
+  hasActive: boolean,
+};
+
+export const Header: React.FC<Props> = ({
+  onAddTodo,
+  todoTitle,
+  setTodoTitle,
+  isBeingAdded,
+  onStatusAll,
+  hasActive,
+}) => {
+  const handleSubmit = (event: FormEvent) => {
+    event.preventDefault();
+
+    onAddTodo(todoTitle);
+  };
+
+  const newTodoField = useRef<HTMLInputElement>(null);
+
+  useEffect(() => {
+    if (newTodoField.current) {
+      newTodoField.current.focus();
+    }
+  }, [isBeingAdded]);
+
+  return (
+    <header className="todoapp__header">
+      {/* eslint-disable-next-line jsx-a11y/control-has-associated-label */}
+      <button
```

Comment:
https://prnt.sc/Xyc8K-l7oZK5

This button should be visible only when todos.length > 0